README.md

```diff
@@ -26,42 +26,77 @@ cd ~/segway_ws
+        * Solution: change Gradle version to 4.4 (can be changed in File > Project Structure > Project or in `gradle-wrapper.properties`)
+    
+        
+
 ### After Installing Android App onto Loomo ###
 
 
 
 The important steps for connecting to the Loomo are (steps 0-2 are one-time only, steps 1-4 described more here: https://gist.github.com/mfe7/c8b21b38a1574f6c319095415ae9a9e8)
 
 1. Set up an NTP server on your Ubuntu PC in order to sync the Ubuntu PC clock to the Loomo Android clock (very important!!). Sync to an NTP server from the internet, then provide a local NTP server for anyone in the 192.168.42.xxx network (Loomo-to-PC network created when USB is connected in next steps). I can share a `ntp.conf` file if needed.
+    - If you edit `/etc/ntp.conf`, run `sudo service ntp restart` to see changes.
 
 2. Switch Loomo into Developer Mode (Settings > System > Loomo Developer > Developer Mode)
     - (I think this requires an internet connection, so join a wifi network now.)
 
-3. You might need to switch Android into Developer Mode (tap some setting 7 times...look online for instructions).
+3. You might need to switch Android into Developer Mode (tap some setting 7 times...look [online](https://www.howtogeek.com/129728/how-to-access-the-developer-options-menu-and-enable-usb-debugging-on-android-4.2/) for instructions).
 
 Note: For the current version of the ROS node, it's very important that the Loomo *not* be connected to wifi,
 because it'll try to use that interface (wlan0) for ROS msgs.
 
 4. Connect USB cable from Loomo's USB-C port to PC's USB-A port.
 
 5. On Loomo, turn on USB Tethering (Settings > Wireless & Networks > More... > Tethering & portable hotspot > USB tethering)
-On Ubuntu, you should see a popup from NetworkManager saying there's a new Wired Connection. You can rename it to Loomo if you want.
+On Ubuntu, you should see a popup from NetworkManager saying there's a new Wired Connection.
+    - If you want to configure a static IP for your PC in the network or want the Loomo to connect to the same "Wired Connection" every time, on Ubuntu PC, search "Network Connections" (Also can be found by clicking Wifi/Network symbol in top right corner > "Edit Connections...")> Add > Ethernet
+        - Connection name: Loomo (or whatever you want it to be)
+        - Go to "IPv4 Settings" tab
+            - Choose "Manual" in the drop down menu
+            - Under "Addresses", Add
+                - Address: 192.168.42.134 (example)
+                - Netmask: 24 (you can leave Gateway blank)
+            - Routes... > check "use this connection only for resources on its network" (this allows you to preserve internet connection when the Loomo connects via ethernet)
+    - After creating a static IP, add these two lines to your `~/.bashrc` (assuming your PC is the ROS master)
+        - `export ROS_MASTER_URI=http://192.168.42.134:11311/
+        - `export ROS_IP=192.168.42.134`
```

Comment:
are these required? is it possible to leave ROS_MASTER_URI as localhost?

Reply:
You still need the ROS_IP but you're right about the ROS_MASTER_URI